src/pages/groupComparison/GroupComparisonUtils.tsx

```diff
@@ -923,3 +938,119 @@ export function getStatisticalCautionInfo() {
+                }
+            >
+                <button
+                    data-test="AlterationEnrichmentTypeSelectorButton"
+                    className="btn btn-primary btn-sm"
+                    style={{ marginBottom: '10px' }}
+                >
+                    {getButtonNameWithDownPointer('Alteration Types')}
+                </button>
+            </DefaultTooltip>
+            <DefaultTooltip
+                trigger={['click']}
+                placement={'bottomRight'}
+                overlay={
+                    <SettingsMenu
+                        store={store}
+                        infoElement={
+                            <AlterationMenuHeader
+                                studyHasMutations={
+                                    store.hasMutationEnrichmentData
+                                }
+                                studyHasStructuralVariants={
+                                    store.hasStructuralVariantData
+                                }
+                                studyHasCnas={store.hasCnaEnrichmentData}
+                                showDriverSection={
+                                    store.showDriverAnnotationMenuSection
+                                }
+                                showTiersSection={
+                                    store.showTierAnnotationMenuSection
+                                }
+                            />
+                        }
+                        customDriverSourceName={
+                            AppConfig.serverConfig
+                                .oncoprint_custom_driver_annotation_binary_menu_label ||
+                            'undefined'
+                        }
+                        showDriverAnnotationSection={
+                            store.showDriverAnnotationMenuSection
+                        }
+                        showTierAnnotationSection={
+                            store.showTierAnnotationMenuSection
+                        }
+                    />
+                }
+            >
+                <button
+                    data-test="AlterationEnrichmentAnnotationsSelectorButton"
+                    style={{
+                        marginLeft: '10px',
+                        marginBottom: '10px',
+                    }}
+                    className="btn btn-primary btn-sm"
+                >
+                    {getButtonNameWithDownPointer('Annotations')}
+                </button>
+            </DefaultTooltip>
+        </div>
+    );
+});
+
+const AlterationMenuHeader: React.FunctionComponent<{
+    studyHasMutations: boolean;
+    studyHasStructuralVariants: boolean;
+    studyHasCnas: boolean;
+    showDriverSection: boolean;
+    showTiersSection: boolean;
+}> = observer(
+    ({
+        studyHasMutations,
+        studyHasStructuralVariants,
+        studyHasCnas,
+        showDriverSection,
+        showTiersSection,
+    }) => {
+        const sections = [];
```

Comment:
i'm generally against this kind of shorthand conditional stuff because i think it's hard to ready, easy to confuse. but we don't have to change it now since i know we want to get this merged

Reply:
Updated nevertheless :)